api/user.json

```diff
@@ -98,5 +153,34 @@
     "trilliumStudentResidence": null,
     "trilliumEnergyAmount": 0,
     "trilliumLongTermCareAmount": 0
+    },
+  "refund": {
+    "Total Payable": {
+      "name": "Total Payable",
+      "line": 435,
+      "amount": 0.00
+    },
+    "Total Deducted": {
+      "name": "Total Deducted",
+      "line": 437,
+      "amount": 0.00
+    },
+    "Total Credits":  {
+      "name": "Total Credits",
+      "line": 482,
+      "amount": 2500.00
+    },
+    "Total Refund": {
+      "name": "Total Refund",
+      "line": 484,
+      "amount": 3200.00
+    }
+  },
+  "benefits": {
+    "Canada child benefit": 6639.00,
+    "Climate action incentive": 0.00,
```

Comment:
Same again here, if @sboisvert's #68 PR goes in, you might have to update the logic here. (Also, I think by default we are giving people Climate Buxx $$$)